source/PlayerInfo.h

```diff
@@ -310,9 +312,6 @@ class PlayerInfo {
 	// Check for and apply any punitive actions from planetary security.
 	void Fine(UI *ui);
 
-	// Set the flagship (on departure or during flight).
-	void SetFlagship(Ship &other);
```

Comment:
Why do you move this? And why only in the .h?

Reply:
because I needed it public

Reply:
Well if you move it in .h you gotta move it in .cpp
And idk about making this public, doesnt seem that clean to me as a solution... Mb a new method that is called when your flagship dies, and would set the flagship to the core ship?

Reply:
Actually nvm about a new function, but do move this in the cpp as well plz

Reply:
done